### ISSUE TITLE
Feature overview for Coin.Dance

### ISSUE THREAD

**Issue by 5chdn** [paritytech/parity-bitcoin#455]
Hey, would love to update the coin.dance entry. Could you help me to understand what's implemented in pbtc?

- [ ] Segregated Witness (SegWit)
- [ ] BIP 101 (BIP 101)
- [ ] 2MB+ Blocks (2MB+ Blocks)
- [ ] Opt-in Full Replace-by-Fee Signaling (RBF)
- [ ] libsecp256k1 (libsecp256k1)
- [ ] getutxos (BIP 64)
- [ ] Includes many new features that have not yet well tested. Use at your own risk! (EXPERIMENTAL)
- [ ] Stores much more information and indexes more data than typical nodes. Therefore, it requires much more space to store the blockchain. (Heavy Client)
- [ ] Makes it possible to run much richer queries that would otherwise be difficult with JSON-RPC-based nodes. (Advanced API)
- [ ] Tracks network consensus rules, allowing a free market to emerge. (Emergent Consensus)
- [ ] Thin Blocks (Thin Blocks)
- [ ] Two million byte size limit with sigop and sighash limits (BIP 109)
- [ ] Differing from other thinblock strategies, “Xtreme Thinblocks” uses simple bloom filtering and a new class of transactions [...] A typical 1MB block can be reduced to a 10KB to 25KB thinblock (Xthin Blocks)
- [ ] Easily configure how much bandwidth should be used, allowing the client to run unobtrusively in a home network. (Traffic-shaping)
- [ ] Relays double spends when they are seen, so sellers can learn about attempts to defraud them faster. (Double spend relaying)
- [ ] A change to the semantics of the 'version' field in Bitcoin blocks, allowing multiple backward-compatible changes (further called &quot;soft forks&quot;) to be deployed in parallel. (Version bits w/ timeout &amp; delay)
- [ ] Relative lock-time using consensus-enforced sequence numbers (BIP 68)
- [ ] A new opcode for the Bitcoin scripting system that, in combination with BIP 68, allows execution pathways of a script to be restricted based on the age of the output being spent. (CHECKSEQUENCEVERIFY)
- [ ] Median time-past as endpoint for lock-time calculations (BIP 113)
- [ ] Peer nodes can be pre-configured for single-hop Xthin block propagation and faster block relay by removing handshake messaging. (Xpedited Block Forwarding)
- [ ] Seeding the bloom filter for Xthin with a small subset of the memory pool where the set is based upon the most probable to-be-mined transactions. (Targeted Bloom Filters)
- [ ] Runs via JavaScript in a web browser. (Runs in-browser)
- [ ] Compact blocks on the wire as a way to save bandwidth for nodes on the P2P network. (Compact Blocks)
- [ ] Flexible Transactions (Flexible Transactions)
- [ ] (BIP 151)
- [ ] JSON RPC API- bitcoind compatible (JSON RPC API)
- [ ] Discards blockchain data when it is no longer needed (Automatic blockchain pruning)
- [ ] Enables the user to control discarding of no-longer-needed blockchain data (Manual blockchain pruning)
- [ ] GUI options to configure node policy (User-friendly node policy configuration)
- [ ] Allows the user to configure how their node filters spam (Configurable spam filtering)
- [ ] (ZeroMQ notifications)
- [ ] Allows the user to easily monitor new blocks and transactions as seen by their node (Network Watch tool)
- [ ] Enables the user to increase the fee on their unconfirmed transactions (Transaction fee bumping)
- [ ] Spam filtering support which cannot be turned off or configured (Mandatory spam filters)
- [ ] BIP 148: Mandatory activation of segwit deployment (User Activated Soft Fork, BIP 148)
- [ ] Extension Blocks
- [ ] SegWit2x
- [ ] Adjustable Blocksize Cap
